Dealership Homework/Dealership/src/main/java/ro/siit/Manufacturer.java

```diff
@@ -0,0 +1,13 @@
+package ro.siit;
+
+/**
+ * Created by Dani on 25-Aug-17.
+ */
+public class Manufacturer extends Dealership {
```

Comment:
There is no inheritence relationship between these two. Try the questions: is-a, has-a. Does any of them apply?